examples/generic_connector_configs/twitter_secretless.yml

```diff
@@ -14,3 +14,28 @@ services:
       forceSSL: true
       authenticateURLsMatching:
         - ^http[s]*\:\/\/api\.twitter\.com*
+  twitter-oauth1:
+    connector: generic_http
+    listenOn: tcp://0.0.0.0:8061
+    credentials:
+      consumer_key:
+        from: keychain
+        get: service#twitter/consumer-key
+      consumer_secret:
+        from: keychain
+        get: service#twitter/consumer-secret
+      token:
+        from: keychain
+        get: service#twitter/token
+      token_secret:
+        from: keychain
+        get: service#twitter/token-secret
+    config:
+      forceSSL: true
+      oauth1Params:
+        oauth_consumer_key: "{{ .consumer_key }}"
+        oauth_consumer_secret: "{{ .consumer_secret }}"
+        oauth_token: "{{ .token }}"
+        oauth_token_secret: "{{ .token_secret }}"
```

Comment:
`oauth_` prefix seems superfluous since we're already in oauth1 section of config